### DISCUSSION TITLE
Shinigami: Community Calls

### DISCUSSION THREAD

**Discussion by b-j-roberts** [keep-starknet-strange/shinigami#94] [Announcements]
## Community Calls 🍎 

Shinigami is a community project focused on being developed in the open with open-source contributors.

This discussion will be used to coordinate Community Calls, in which community members can join to learn more about the project and its short and long-term goals. Anyone is welcome to join, express their concerns and opinions, or listen in to the latest Shinigami updates!

Thanks!

**Comment by b-j-roberts** [comment]
### Shinigami: Community Call 1 🍎 

**August 2nd (Friday) @ 3PM UTC** : https://starkware-co.zoom.us/j/89627730582

The first Shinigami community call will be held over Zoom and be recorded for anyone who cannot make it.

In this call we will try and cover:
- Short introductions to the team/community members
- The project summary and goals
- Intro to Bitcoin Script and Shinigami's design
- What has been done and what is to come

**Reply by adrienlacombe** [reply]
could be nice to get a calendar invite :)

**Reply by varun-doshi** [reply]
Yes calendar invite would be helpful

**Reply by b-j-roberts** [reply]
A little late but, calendar invite: https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NHVucDI0ZXM2NTFmcGZhZzBuOHRpazMzM2sgYnJhbmRvbkBzdGFya3dhcmUuY28&tmsrc=brandon%40starkware.co

**Comment by b-j-roberts** [comment]
Shinigami: Community Call 2 🍎

**August 28th (Friday) @ 5PM UTC** : [Zoom link](https://starkware-co.zoom.us/j/86790694858) -- [Calendar Link](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MjFrNWZ2OHVyZzg3MzlucDczZm90bGRqM2cgYnJhbmRvbkBzdGFya3dhcmUuY28&tmsrc=brandon%40starkware.co)

The second Shinigami community call will be held over Zoom and be recorded for anyone who cannot make it.

In this call, we will try and cover:

- Project Updates: Web-app, Opcodes, transaction support, hashing, signatures, ...
- What's next: Prover integration and proving Bitcoin mainnet
- Knowledge Sharing: P2PK, P2PKH, Coinbase transactions, and more
- Open discussion: Brainstorming, questions, thoughts, after shinigami?
